cdap-watchdog/src/main/java/co/cask/cdap/logging/read/AvroFileReader.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2014-2016 Cask Data, Inc.
```

Comment:
I just copied the copyright template over from https://wiki.cask.co/display/CE/Setting+up+environment#Settingupenvironment-SettingupautoCopyrightheaderinIntelliJ